packages/ketchup/src/components/kup-activity-timeline/kup-activity-timeline.scss

```diff
@@ -0,0 +1,176 @@
+.atm-container {
+  color: var(--kup-gray-color-70);
+}
+.atm-container.clickable {
```

Comment:
To improve readability of the CSS rules, you should use scss syntax (nesting), inline selectors such as "atm-container .atm-event .atm-thumb > span.-atm-icon" cause confusion.
Refer to `packages\ketchup\src\components\kup-accordion\styles\kup-accordion-main.scss` for a neat example.